src/datasource/OGC.js

```diff
@@ -476,6 +476,22 @@ class OGC extends ngeoDatasourceDataSource {
     return this.dimensions_;
   }
 
+  /**
+   * @return {string} Filter condition
+   * @export
+   */
+  get filterCondition() {
```

Comment:
why do we need to have getter and setter for this variable ?

Reply:
1) This variable needs to be changed.  In the filter component, one can set OR instead of AND, which affect this variable, thus it needs a setter
2) It needs a getter because we bind UI components to it
3) If you look around the other properties of the data source classes, you'll notice that they have getters/setters as well and those we're removed.  If this one was removed, then why not the others?
4) Back then, when this was first implemented, it was my first time writing ES6 classes, and the use of getters/setters seemed quite nice and it was working with Angular well.  This was a good choice.
5) The typecheck complains that there is not a filterCondition property set.  Is this because inner properties can't be accessed?

Please, comment.  Thanks.

Reply:
ok, so it was changed ("simple" public variable to getter / setter) for the typecheck ?

Reply:
I don't know why it was changed.  It was using a getter/setter before and it was working fine.